reactioncommerce/reaction
# [WIP] Refactor 3570 willopez eslint rules

Resolves #3570 

Fixes eslint errors.

**Test instructions**

- Run eslint and confirm no errors are displayed.
- Run tests and confirm there are no errors.
